torchqdynamics/ode/odeint.py

```diff
@@ -12,14 +13,28 @@
 # --------------------------------------------------------------------------------------
 
 
+class Solver(Enum):
+    DOPRI5 = 'dopri5'
+    RK = 'rk'
+    OUT = 'out'
+
+
 def odeint(
-    f, y0, tspan, solver='dopri5', save_at=(), sensitivity=None, model_params=None,
-    atol=1e-8, rtol=1e-6, backward_mode=False
+    f,
+    y0,
+    tspan,
+    solver=Solver.DOPRI5,
```

Comment:
The end user should be supplying a string as the `odeint` argument, and not an object. It's much simpler on the user end since there's no need to import an additional object.

Reply:
So we need to convert afterwards from the string to the Solver argument afterwards